src/browser/connection/index.ts

```diff
@@ -58,9 +60,12 @@ interface ProviderMetaInfoOptions {
     appendToUserAgent?: boolean;
 }
 
+type BrowserOption = FireFoxConfig | ChromeConfig | string;
```

Comment:
I feel it's better to mark it as `unknown`. Since third-party providers can have different config formats in the future.